t/t0028-encoding.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+test_description='encoding conversion via gitattributes'
+
+. ./test-lib.sh
+
+test_expect_success 'setup test repo' '
+
+	text="hallo there!\ncan you read me?" &&
+
+	echo "*.utf16 text encoding=utf-16" >.gitattributes &&
```

Comment:
The spacing around `>` redirects is inconcistent in this test file. I’d personally put spaces before and after each `>`.

Reply:
No space after `>` is Git style 😉

Reply:
Okay, but below there are cases where there is no space before `>`, and here, there is one.